packages/plugins/experimental/plugin-wnfs/src/components/FileMain.tsx

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React, { type FC } from 'react';
+
+import { Main } from '@dxos/react-ui';
+import { baseSurface, topbarBlockPaddingStart, fixedInsetFlexLayout } from '@dxos/react-ui-theme';
+
+import { FilePreview } from './FilePreview';
+import { type FileType } from '../types';
+
+const FileMain: FC<{ file: FileType }> = ({ file }) => {
+  const url = 'TODO';
```

Comment:
?

Reply:
I haven't worked on these components, was planning to do that in a separate PR. I'd also like some input on how to proceed with these components.

Reply:
@richburdon See the comment above.

Reply:
ok we're doing a rewrite of mosaic, so i think fine to leave this.